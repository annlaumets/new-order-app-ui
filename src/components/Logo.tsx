import { AspectRatio, AspectRatioProps } from '@chakra-ui/react'
import Image from 'next/image'

const Logo: React.FC<AspectRatioProps> = ({ ...props }) => {
  return (
    <AspectRatio ratio={237 / 56} position="relative" {...props}>
      <Image src="/images/logos/logo.svg" alt="frogs-logo" layout="fill" />
    </AspectRatio>
  )
}

export default Logo
