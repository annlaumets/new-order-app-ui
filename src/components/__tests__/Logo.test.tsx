import { render } from '@testing-library/react'

import Logo from 'components/Logo'

describe('Logo.tsx component', () => {
  const { container } = render(<Logo />)

  test('match to snapshot', () => {
    expect(container).toMatchSnapshot()
  })
})
