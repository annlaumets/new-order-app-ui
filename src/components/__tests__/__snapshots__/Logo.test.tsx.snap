// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo.tsx component match to snapshot 1`] = `
<div>
  <div
    class="chakra-aspect-ratio css-2xipno"
  >
    Next image stub
  </div>
</div>
`;
